merlin/dag/base_operator.py

```diff
@@ -20,6 +20,7 @@
 from typing import Any, List, Optional, Union
 
 import merlin.dag
+import merlin.dag.utils
```

Comment:
Unless I'm missing something, this import now seems unnecessary?

Reply:
good call